src/styles/index.css

```diff
@@ -1,295 +1,435 @@
 .home-page {
-    max-width: 960px;
```

Comment:
Need to force push to fix:

- [ ] Remove lint noise
- [ ] Move `.node-demo` styles into [src/styles/demo-tabs.css](https://github.com/nodejs/nodejs.dev/pull/426/files#diff-59ed09e784401f4fc115b27dd19d474b)